src/app/submit-email-template/[sessionId]/page.tsx

```diff
@@ -7,9 +7,9 @@ import { getSession } from '@utils/form-actions';
 
 const isValid = (session?: Session) =>
   session?.templateType === TemplateType.EMAIL &&
-  session?.emailTemplateName !== undefined &&
-  session?.emailTemplateSubjectLine !== undefined &&
-  session?.emailTemplateMessage !== undefined;
+  Boolean(session?.emailTemplateName) &&
```

Comment:
Amplify will return empty fields as `null` not undefined. So this check wasn't working as expected.